### ISSUE TITLE
Syntax coloring issue

### ISSUE THREAD

**Issue by KrunchMuffin** [atuttle/atom-language-cfml#77]
I have an href that points to a div like so `<a href="#EditForm" id="NewAdjust" class="btn blue btn-sm"><i class="fa fa-pencil"></i> New Adjustment</a>`
I am not inside a cfoutput so I have no need to double it up. This is breaking the syntax coloring.

![image](https://cloud.githubusercontent.com/assets/1037197/12720932/2c472140-c8cb-11e5-878a-411ded27ff23.png)

**Comment by scottyinzeo** [comment]
+1

**Comment by infernocloud** [comment]
Yeah this has been happening for awhile unfortunately. I haven't heard much from the maintainers lately either, starting to worry.

**Comment by KrunchMuffin** [comment]
I have to stick with Sublime for now until this matures.

**Comment by atuttle** [comment]
@infernocloud I'm still here any do check on every ticket that comes in. Unfortunately I don't have the spare time to learn the grammar-definition syntax, and my motivation is low because I still use Sublime as my "daily driver."

I created this package as a mechanical port (the Atom team provides a tool to port Textmate Bundles, and I used it) when I got access to Atom, because there was no CFML package. I try to maintain an active role in merging pull requests and publishing releases, but that's the extent of my capability at the moment.

I'm happy to grant commit access to anyone that wants to help out, though. @seancoyne and @elpete have done a tremendous job building up some test cases and resolving a bunch of issues. More than my contribution, by far. :beers:
